server/controllers/v1/tests/comment.test.ts

```diff
@@ -25,19 +25,19 @@ describe('Test v1 - Comment Controller', () => {
         await makeValidComment(author.id, post.id),
       ];
       const result = await commentController.getComments(post.id, 100, 0);
-
       expect(result.status).toBe(200);
-      for (let index = 0; index < 3; index++) {
-        expect(result.data.result![index].id).toBe(comments[index].id);
-        expect(result.data.result![index].User.firstName).toBe(
-          author.firstName
-        );
-      }
+      expect(result.data.result!.map((x) => x.id).sort()).toEqual(
+        comments.map((x) => x.id).sort()
+      );
+      expect(
+        result.data.result!.map((x) => x.User.firstName === author.firstName)
+      ).toBeTruthy();
```

Comment:
```suggestion
      expect(result.data.result!.map((c) => c.id).sort()).toEqual(
        comments.map((c) => c.id).sort()
      );
      expect(
        result.data.result!.map((c) => c.User.firstName === author.firstName)
      ).toBeTruthy();
```
`c` for comment, we're not in math lol